pkg/views/label/list/view.go

```diff
@@ -0,0 +1,79 @@
+package list
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/models"
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/goharbor/harbor-cli/pkg/views"
+)
+
+type model struct {
+	table table.Model
+}
+
+var columns = []table.Column{
+	{Title: "ID", Width: 6},
+	{Title: "Name", Width: 12},
+	{Title: "Color", Width: 12},
+	{Title: "Description", Width: 18},
+	{Title: "Creation Time", Width: 24},
+}
+
+func (m model) Init() tea.Cmd {
```

Comment:
Just use base styling, there already the code in place to use that.

Reply:
@amands98, Thanks for the correction, I will change it accordingly